src/Store/index.js

```diff
@@ -52,22 +49,18 @@ const middleware = (getDefaultMiddleware) =>
     notificationsMiddleware({
       errorTitleKey: ['message'],
       errorDescriptionKey: ['response.data.detail'],
-    }),
-    production && logger
```

Comment:
That the store is memoized is fine and the `useMemo` can and maybe should stay, but this here seems to be the only line that makes a difference and resolves the error. 

However, I think instead of removing this we should rather be fixing it and enable redux logging properly.

Other apps pass in the the logger depending what entry they are coming from, like here: https://github.com/RedHatInsights/compliance-frontend/blob/master/src/DevAppEntry.js#L5

We can take a similar approach here and avoid using the `env/production` consts here. 

Another thing that compliance does, that may be relevant here is it filters out undefined middlewares passed in. This may help here as well to prevent the issues we were seeing.

Reply:
I do not think that memoization of the store is good practice. The useMemo is not designed for it, I think. Rather, it is [recommended](https://redux-toolkit.js.org/tutorials/quick-start#provide-the-redux-store-to-react) way to assign the initialized store to some constant variable. If the store is a constant var outside of react, it will not be reinitilised on every re-render. This is what Karel wanted to fix with useMemo

Reply:
Still, we should rather fix the passing of the logger, than just removing it, because it is useful in dev.

Reply:
Oh, yes. I will bring it back.

Reply:
I think we'll also need to properly init the logger. See: https://www.npmjs.com/package/redux-logger#usage